packages/react/src/components/Forms/Input/index.tsx

```diff
@@ -96,14 +96,12 @@ export const Input = forwardRef<InputRefType, Props>(
             "c__input__wrapper--" + state,
             {
               "c__input__wrapper--disabled": props.disabled,
-              "c__input__wrapper--focus": inputFocus,
             }
           )}
           onClick={() => {
             inputRef.current?.focus();
           }}
-          tabIndex={0}
-          role="textbox"
+          role="presentation"
```

Comment:
A `div` tag has already no semantic. So `role=presentation` is useless here. Furthermore, I'm not quit sure to understand the whole concept, but a role="presentation" can also remove semantic of its descendant that is not something we want.